Automattic/wp-calypso
# AnchorFM: Podcasting Card

#### Changes proposed in this Pull Request

- Added a task card for starting a podcast website with Anchor.fm

#### Testing instructions

1. Sandbox the API and apply this diff D52847-code
2. Uncomment line 54 from `wp-content/lib/home/views.php`
3. Use Calypso Live to checkout this branch: https://github.com/Automattic/wp-calypso/pull/47487
4. On your site https://wordpress.com/home/{site}, where you've already gone through the introductory set-up tasks, dismiss the cards at the top until you see the podcast card.
5. Click on "Get Started". This should take you to https://anchor.fm/. Clicking on "Hide this" should dismiss this card.

#### Screenshots and GIFs

<img width="1001" alt="Screen Shot 2020-11-16 at 7 29 10 PM" src="https://user-images.githubusercontent.com/66652282/99339451-31097780-2854-11eb-92e9-49502503c162.png">

![2020-11-16 19 29 15](https://user-images.githubusercontent.com/66652282/99339471-3c5ca300-2854-11eb-9d58-813d01f3eb01.gif)

![2020-11-16 19 29 28](https://user-images.githubusercontent.com/66652282/99339477-3d8dd000-2854-11eb-879c-66a6bde74f76.gif)


Fixes 310-gh-dotcom-manage
